packages/pages/src/components/breadcrumbs/methods.test.ts

```diff
@@ -0,0 +1,32 @@
+import { defaultTransformer } from "./methods.js";
+import { StreamsBreadcrumbs } from "./yextBreadcrumbs.js";
+
+function doTestDefaultTransformer() {
+  const testDetails: Array<[StreamsBreadcrumbs, Array<[string, string]>]> = [
+    [
+      {
+        dm_directoryParents: [
+          { slug: "index.html", name: "All locations" },
```

Comment:
This raw data seems like it would be fine to use without transforming into the tuple